packages/headless/src/controllers/product-listing/facet/headless-product-listing-facet.ts

```diff
@@ -62,34 +67,33 @@ export function buildFacet(
     ...coreController,
 
     toggleSelect: (selection: FacetValue) => {
-      dispatch(
-        executeToggleFacetSelect({
-          facetId: getFacetId(),
-          selection,
-        })
-      );
+      coreController.toggleSelect(selection);
       dispatch(fetchProductListing());
       dispatch(getAnalyticsActionForToggleFacetSelect(getFacetId(), selection));
     },
 
     deselectAll() {
       coreController.deselectAll();
       dispatch(fetchProductListing());
+      dispatch(logFacetClearAll(getFacetId()));
```

Comment:
I noticed these were missing.

Even though we do server-side logging, I'm not sure we'll be tracking individual facet actions, so for now, we'll log them. :thinking: